doc/release-notes.md

```diff
@@ -23,492 +26,644 @@ shut down (which might take a few minutes for older versions), then run the
-future version.
-
-Mempool sync improvements
--------------------------
-Nodes joining the network will now try to sync their mempool from other v0.15+ peers via the `mempool` p2p message.
-This behaviour can be disabled via the new `--syncmempool` option. Nodes serving such requests will now also push
-`inv` p2p messages for InstandSend locks which are held for transactions in their mempool. These two changes
-should help new nodes to quickly catchup on start and detect any potential double-spend as soon as possible.
-This should also help wallets to slightly improve UX by showing the correct status of unconfirmed transactions
-locked via InstandSend, if they were sent while the receiving wallet was offline. Note that bloom-filters
-still apply to such `inv` messages, just like they do for transactions and locks that are relayed on a
-regular basis.
-
-PrivateSend improvements
-------------------------
-This version decouples the so called "Lite Mode" and client-side PrivateSend mixing, which allows client-side mixing
-on pruned nodes running with `--litemode` option. Such nodes will have to also specify the newly redefined
-`--enableprivatesend` option. Non-prunned nodes do not have to do this but they can use `--enableprivatesend`
-option to disable mixing completely instead. Please note that specifying this option does not start mixing
-automatically anymore (which was the case in previous versions). To automatically start mixing, use the new
-`--privatesendautostart` option in addition to `--enableprivatesend`. Additionally, PrivateSend can always be
-controlled with the `privatesend` RPC.
-
-Thanks to LLMQ-based InstantSend and its ability to lock chains of unconfirmed transactions (and not only a single
-one like in the legacy system), PrivateSend mixing speed has improved significantly. In such an environment
-Liquidity Provider Mode, which was introduced a long time ago to support mixing volume, is no longer needed and
-is removed now. As such the `--liquidityprovider` option is not available anymore.
-
-Some other improvements were also introduced to speed up mixing, e.g. by joining more queues or dropping potential
-malicious mixing participants faster by checking some rules earlier etc. Lots of related code was refactored to
-further improve its readability, which should make it easier for someone to re-implement PrivateSend
-correctly in other wallets if there is a desire to do so.
-
-Wallet changes
---------------
-Wallet internals were optimized to significantly improve performance which should be especially notable for huge
-wallets with tens of thousands of transactions or more. The GUI for such wallets should be much more responsive too now.
+Two new sporks were introduced in this version:
+- `SPORK_21_QUORUM_ALL_CONNECTED` allows to control Concentrated Recovery and
+Masternode Probing activation;
+- `SPORK_22_PS_MORE_PARTICIPANTS` rises the number of users that can participate
+in a single PrivateSend mixng transaction to 20.
```

Comment:
If I remember, 0.16 also changes mixing from being a random number of participants between 3 and 5 to being as many participants as are available with a minimum of 3 and a max of 20